crates/dpapi-core/src/str.rs

```diff
@@ -27,11 +29,11 @@ pub fn from_utf16_le(data: &[u8]) -> Result<String> {
 /// Encodes str into a UTF-16 encoded byte array.
 ///
 /// *Note*: this function automatically appends a NULL-char.
-pub fn encode_utf16_le(data: &str) -> Vec<u8> {
+/// *Panic*: panics when cursor's internal buffer doesn't have enough space.
```

Comment:
note: Typically this kind of information is specified in a dedicated section. See: https://rust-lang.github.io/api-guidelines/documentation.html#function-docs-include-error-panic-and-safety-considerations-c-failure